Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -914,10 +914,18 @@ private extension PurchasesOrchestrator {
             case let .failure(error):
                 let purchasesError = error.asPublicError
 
+                #if swift(>=5.6)
                 @MainActor
                 func complete() {
                     completion?(transaction, nil, purchasesError, false)
                 }
+                #else
+                @Sendable
+                @MainActor
+                func complete() {
```

Comment:
can't wait until April when Xcode 13 is dropped and we can come back to kill this bit